lib/view/reusable_infographic_multipage.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+import 'package:photo_view/photo_view.dart';
+import 'package:photo_view/photo_view_gallery.dart';
+
+import '../style.dart';
+
+class InfographicMultiPage extends StatefulWidget {
+  // Navigation bar title
+  final String title;
+  final List<String> imageUrls;
+  final Color backgroundColor;
+  final Color toolBarColor;
+
+  const InfographicMultiPage({
+    this.title,
+    this.imageUrls,
+    this.backgroundColor,
+    this.toolBarColor,
+  });
+
+  @override
+  _InfographicMultiPageState createState() => _InfographicMultiPageState();
+}
+
+class _InfographicMultiPageState extends State<InfographicMultiPage> {
+  int _pageIndex = 0;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        backgroundColor: widget.backgroundColor,
+        appBar: AppBar(
+          // Warning brightness interacts with SystemUiOverlayStyle
+          // See system_bars.dart comments
+          brightness: Brightness.light,
+          backgroundColor: widget.toolBarColor,
+          iconTheme: Styles.appBarIconTheme,
+          title:
+              Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
+            Text(
+              widget.title,
+              style: Styles.textH5,
+            ),
+            Text(
+              'Page ${_pageIndex + 1} of ${widget.imageUrls.length}',
+              style: Styles.textP,
+            )
+          ]),
+          leading: IconButton(
+            icon: Icon(Icons.close),
+            onPressed: () => Navigator.of(context).pop(null),
+          ),
+        ),
+        body: PhotoViewGallery(
+          scrollPhysics: const BouncingScrollPhysics(),
```

Comment:
Not so sure about bouncing on overscroll, because all of the other screens use the platform default. So all overscrolls will bounce on iOS anyway, but Android will be the overscroll 'glow' like all other screens. Looks ok without this ... if a bit 'light' in color.

Perhaps on Android can darken the color a little to make it stand out more with **GlowingOverscrollIndicator**() ? Or I'm sure default color is also in the theme somewhere.

Reply:
> I checked out the branch and gave it some testing - unfortunately once you have zoomed in, you can't pan down to view other parts of the document:
> 
> ![Apr-29-2020 19-39-21](https://user-images.githubusercontent.com/902252/80581865-2ec70380-8a51-11ea-9e77-f8fdd3699237.gif)
> 
> In this gif I was trying to zoom in and pan down to view the table at the bottom of the 1st page, but unfortunately you cant do it.

This works for me, so I think it depends on the aspect ratio of your screen. It's probably a bug in the `photo_view` library

Reply:
Removed bouncy scroll